python/mxnet/ndarray.py

```diff
@@ -1224,6 +1226,12 @@ def add(lhs, rhs):
     >>> (x+y).asnumpy()
     array([[ 1.,  1.,  1.],
            [ 2.,  2.,  2.]], dtype=float32)
+    >>> mx.nd.add(x,y).asnumpy()
+    array([[ 1.,  1.,  1.],
+           [ 2.,  2.,  2.]], dtype=float32)
+    >>> mx.nd.broadcast_add(x,y).asnumpy()
+    array([[ 1.,  1.,  1.],
+           [ 2.,  2.,  2.]], dtype=float32)
```

Comment:
All three of these examples return the same thing - and two use different functions than the one being documented here. What precisely do you intent to show? And what are the initial values of x and y? Why isn't this indicated as part of the example? I think examples should be self-contained.